src/views/SignTransactionSuccess.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
Basically a copy of `CheckoutSuccess.vue` with only minor wording and button label differences. In the future, `SignTransaction` and `Checkout` flows are potentially handled more different though (sending tx to central server or directly into network during checkout, as opposed to returning it to the Safe for a regular signature request), thus it makes sense to separate the two.